YCloudYUSA/yusaopeny
# feat: Add new demo content

WiP: https://github.com/YCloudYUSA/y_lb_demo_content/pull/7

Original Issue, this PR is going to fix: REPLACE WITH A LINK TO ISSUE ( publicly available )


Make sure these boxes are checked before asking for review of your pull request - thank you!

If there is a new feature or this is a bug fix - use 9.x-2.x branch. We'll tag for release if the bug is critical asap or tag for release next bug fix release until critical issue arrived.

## Steps for review

 - go to admin/modules
 - enable the Y Layout Builder - Init Demo Pages module
 - go to /admin/structure/migrate/manage/y_lb_init_demo_content/migrations
 - run this migration /admin/structure/migrate/manage/y_lb_init_demo_content/migrations/y_lb_init_demo_media_image/execute
 - run these migrations : 
    -  /admin/structure/migrate/manage/y_lb_init_demo_content/migrations/y_lb_init_demo_homepage/execute
    - /admin/structure/migrate/manage/y_lb_init_demo_content/migrations/y_lb_init_demo_page_who_we_are/execute
    - /admin/structure/migrate/manage/y_lb_init_demo_content/migrations/y_lb_init_demo_page_programs_overview/execute
    - /admin/structure/migrate/manage/y_lb_init_demo_content/migrations/y_lb_init_demo_page_programs_category/execute
 -  as alternative you can run this drush command: `drush migrate:import --group y_lb_init_demo_content`
 - go to /admin/content
 - visit all page with prefix in the title " Init Demo"
 - verify that pages looks like on design

- [ ] Check http://profile.openy.cibox.tools/build4428/demo-layout-builder-page-home
![image](https://github.com/YCloudYUSA/yusaopeny/assets/563412/982bda51-3949-4908-a749-83b2b5bd29f9)




## General checks
- [ ] [Documentation](https://github.com/ymcatwincities/openy/tree/9.x-2.x/docs) has been updated according to PR changes.
- [ ] [Steps for review](https://github.com/ymcatwincities/openy/pull/94#issue-204580200) have been provided according to PR changes. <br/><img src="https://raw.githubusercontent.com/ymcatwincities/openy/9.x-2.x/.github/assets/steps-for-review.png" width="200" alt="Steps for review"/>
- [ ] Make sure you've provided all necessary hook\_update\_N to [support upgrade path](https://github.com/ymcatwincities/openy/blob/9.x-2.x/docs/Development/Upgrade%20path.md).
- [ ] Make sure your git email is associated with account on drupal.org, otherwise you won't get commits there. <br/><img src="https://raw.githubusercontent.com/ymcatwincities/openy/9.x-2.x/.github/assets/drupalorg-email.png" width="200" alt="drupal.org email"/>
- [ ] If you would like to get credits on drupal.org, [check documentation](https://github.com/ymcatwincities/openy/blob/9.x-2.x/docs/Development/Contributing.md#drupalorg-credits).

Thank you for your contribution!
